sprocs/instance_questions_update_for_manual_grading.sql

```diff
@@ -0,0 +1,51 @@
+DROP FUNCTION IF EXISTS instance_questions_update_for_manual_grading(bigint, bigint, bigint);
+
+-- RETURN next instance question for grading from assessment question ID
+
+CREATE OR REPLACE FUNCTION
+    instance_questions_update_for_manual_grading(
+        IN arg_assessment_id bigint, -- endpoint auth redundancy
+        IN arg_assessment_question_id bigint,
+        IN arg_user_id bigint,
+        OUT instance_question jsonb
+    )
+AS $$
+DECLARE
+    iq_id bigint;
+BEGIN
+
+    -- Get LAST submissions, filter for ungraded entries to find next ungraded instance question
+    SELECT iq.id
+        FROM instance_questions AS iq
+            JOIN variants AS v ON (v.instance_question_id = iq.id)
+            JOIN submissions AS s ON (s.variant_id = v.id)
+    INTO iq_id
+    WHERE (s.auth_user_id, s.date) 
+        IN (
+            SELECT s.auth_user_id, MAX(s.date)
```

Comment:
I still want to change this logic here, but aside from that, everything seems to be working and a lot more simplified. Most of your comments have been overwritten @mwest1066 .